lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -5485,7 +5486,11 @@ void ScriptContext::RegisterPrototypeChainEnsuredToHaveOnlyWritableDataPropertie
             Output::Print(_u("\n\n"));
 
             // If in verbose mode, dump data for each FunctionBody
-            if (Configuration::Global.flags.Verbose && rejitStatsMap != NULL)
+            if (
+#if defined(FLAG) || defined(FLAG_REGOVR_EXP)
```

Comment:
Part of the goal for this change is so that for e.g. crawling, this code can be enabled in non-debug builds.